packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -1,8 +1,8 @@
-const { exec } = require('child_process');
 import chalk from 'chalk';
 import path, { sep } from 'path';
 import { installPackages, lintFix, nextSteps, BaseArgs, saveConfiguration } from './common';
 import { InitializerFactory } from './InitializerFactory';
+import { installPrePushHook } from './common/processes/install';
```

Comment:
Should this be exported/imported from ./common index (same as installPackages, lintFix)?